### ISSUE TITLE
Introduction: include an example pair of documents, one Text and one Image profile

### ISSUE THREAD

**Issue by nigelmegitt** [w3c/imsc#553]
In the introduction, it would be _really_ helpful to include an example pair of documents, one Text and one Image profile, that demonstrate what IMSC can be used for, and the kind of expected output. Ideally they would be annotated.

_Originally posted by @nigelmegitt in https://github.com/w3c/imsc/pull/552_

**Comment by palemieux** [comment]
@nigelmegitt Annex E already contains examples. Would a sample render of both resolve your issue?

https://www.w3.org/TR/ttml-imsc1.2/#sample-instance

**Comment by nigelmegitt** [comment]
@palemieux yes, though for preference I would add some styling to the Text profile example, and not just in the `region` - for example a different color and background color applied to the text content by putting it into spans.

**Comment by palemieux** [comment]
There are also the existing [MDN pages](https://developer.mozilla.org/en-US/docs/Related/IMSC).
